src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/ILCompilerRIDs.props

```diff
@@ -14,5 +14,6 @@
     <OfficialBuildRID Include="win-x64" Platform="x64" />
     <OfficialBuildRID Include="freebsd-x64" Platform="x64" />
     <OfficialBuildRID Include="freebsd-arm64" Platform="arm64" />
+    <OfficialBuildRID Include="$(OutputRID)" Platform="$(TargetArchitecture)" Exclude="@(OfficialBuildRID)" />
```

Comment:
What is the purpose of `Exclude="@(OfficialBuildRID)"`? Is it going to remove all `OfficialBuildRID`s that just got added?